app/assets/stylesheets/content/_tables.sass

```diff
@@ -26,14 +26,59 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
+table
+  td
+    padding: 3px 6px
+  p
+    margin: 0
+
+  &.files tr.file td
+    text-align: center
+    &.filename
+      text-align: left
+      padding-left: 24px
+    &.digest
+      font-size: 80%
+
+  &.members td.roles, &.memberships td.roles
+    width: 45%
```

Comment:
The only place where I could see this rule apply was here: http://localhost:5000/users/8/edit/memberships

Still I don't get the purpose...